src/hooks/usePortfolio/usePortfolio.ts

```diff
@@ -91,13 +95,15 @@ async function supplementTokensDataFromNetwork({
 }
 
 export default function usePortfolio({
+  fetch, 
   currentNetwork,
   account,
   useStorage,
   isVisible,
   useToasts,
   getBalances
 }: UsePortfolioProps): UsePortfolioReturnType {
+  const { constants, isLoading } = useFetchConstants({ fetch })
```

Comment:
I think it's better to pass either the `useFetchConstants` here, either the `constants` and `isLoading` params. Let's discuss this offline.